### PR TITLE
Add new step to force reclone of repo

### PR DESCRIPTION
**Pull request by daxmc99**
Related to https://sourcegraph.slack.com/archives/C89KCDK5J/p1663606116675159

Tested on s2

### CODE REVIEW

File: content/departments/cloud/technical-docs/operations.md [hunk 1]
```diff
@@ -374,6 +390,8 @@ Once you have identified a repo is constantly failing to be updated/fetched, exe
    export DEPLOYMENT=$(gcloud compute instances list --project "$PROJECT_PREFIX-$CUSTOMER" | grep -v "executors" | awk 'NR>1 { if ($1 ~ "-red-") print "red"; else print "black"; }')
    ```
 
+````
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/operations.md]: Looks like an oversight? i.e. should we delete it

File: content/departments/cloud/technical-docs/operations.md [hunk 2]
```diff
@@ -549,3 +567,4 @@ This likely means our admin user account was deleted. To verify
 - run `SELECT id,username,deleted_at FROM users ORDER BY created_at LIMIT 1;` and check if the third value is set to a non NULL value (a date)
 - if you see `deleted_at` set, take note of ID (usually equal to 1, returned in first column) and use `UPDATE users SET deleted_at = NULL where ID = ?` (substitute `?` for id) to mark the user as not deleted
 - login should now work correctly
+````
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/operations.md]: Ditto

**Review by unknwon — APPROVED** [review]
Approve to unblock

File: content/departments/cloud/technical-docs/operations.md (hunk 1 above)
Comment (daxmc99) [reply content/departments/cloud/technical-docs/operations.md]: My formatter must have gone a bit haywire

**Review by filiphaftek — COMMENTED** [review]
😍

**Pull request merged**